app/containers/V2FarmingPage/index.js

```diff
@@ -83,8 +84,8 @@ export function FarmingPage(props) {
   useEffect(() => {
     const RGPfarmingFee = async () => {
       if (wallet.signer !== 'signer') {
-        const masterChefV2 = await masterChefV2Contract();
-        const minFarmingFee = await masterChefV2.farmingFee();
+        const masterChef = await masterChefContract();
+        const minFarmingFee = await masterChef.farmingFee();
```

Comment:
Did you accept my changes when you merged, cause this data should be coming from v2